client/src/components/RegisterForm/registerForm.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Wrapper from "../Wrapper/Wrapper";
+import GitHubLoginBtn from "../GitHubLoginBtn/GitHubLoginBtn";
+import GoogleLoginBtn from "../GoogleLoginBtn/GoogleLoginBtn";
+import Joi from "joi-browser";
+import Form from "../Form/form";
+import * as userService from "../services/userService";
+
+// import "./style.css";
+
+class RegisterForm extends Form {
+  state = {
+    data: { firstname: "", lastname: "", email: "", password: "" },
+    errors: {}
+  };
+  schema = {
+    firstname: Joi.string()
+      .required()
+      .label("Firstname"),
```

Comment:
Label is the visible text or the name of the field? If visible should likely be `First Name`